src/main/resources/init_db.sql

```diff
@@ -0,0 +1 @@
+CREATE DATABASE `cinema_app` /*!40100 DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci */ /*!80016 DEFAULT ENCRYPTION='N' */;
```

Comment:
You can remove this file, any way you have already indicated the name of the schema in the hibernate.cfg.xml file